routes/notificationRouter.js

```diff
@@ -1,8 +1,83 @@
-// const express = require('express');
-// const { db } = require('../server/db');
+const express = require('express');
+const { db } = require('../server/db');
 
-// const notificationRouter = express.Router();
+const notificationRouter = express.Router();
 
-// Your code below (delete this line when committing):
+notificationRouter.get('/', async (req, res) => {
+  try {
+    const allNotifications = await db.query('SELECT * FROM notification;');
+    res.status(200).send(allNotifications);
+  } catch (err) {
+    res.status(500).json({
+      status: 'Failed',
+      msg: err.message,
```

Comment:
The way you did it is technically correct, however, it is redundant with indicating that `status: 'Failed'`, do it like so `res.status(500).send(err.message);`.